package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "datadog-cdk-constructs",
-  "version": "0.1.2",
+  "version": "0.1.4",
   "description": "CDK Construct library to automatically instrument python and node functions with datadog tracing",
   "repository": "github.com/DataDog/datadog-cdk-constructs",
   "author": "Datadog",
   "license": "Apache-2.0",
   "bin": {
-    "datadog-cdk-constructs": "src/index.js"
+    "datadog-cdk-constructs": "dist/lib/index.js"
```

Comment:
What is this `bin` for?

Reply:
@nhinsch After messing with CDK some more it seems that when you bootstrap a cdk project it creates the bin config because I believe the bin path is the primary entry point for the cdk project. It seems to be important for the CDK project.

Reply:
[https://cdkworkshop.com/20-typescript/20-create-project/300-structure.html](https://cdkworkshop.com/20-typescript/20-create-project/300-structure.html ) this is what I followed.